components/nav.js

```diff
@@ -29,6 +29,46 @@ const Navbar = glamorous.nav(({top, theme: {mediaQueries}}) => ({
     flexDirection: 'column',
   },
 }))
+// Abstract input primitives to a higher level at a later date
+const inputBase = (props, {colors}) => ({
+  width: 120,
+  borderRadius: 50,
+  paddingTop: 4,
+  paddingBottom: 4,
+  paddingLeft: 30,
+  paddingRight: 20,
+  backgroundImage: 'url(/static/images/search.svg)',
+  // Icons made by Madebyoliver from www.flaticon.com is licensed by CC 3.0 BY
+  // http://www.flaticon.com/authors/madebyoliver
+  backgroundRepeat: 'no-repeat',
+  backgroundPosition: '5px center',
+  backgroundSize: 'auto 80%',
+  border: `1px solid ${colors.primaryMed}`,
+  color: colors.darkGray,
+  ':focus': {
+    width: 200,
+    outline: 'none',
+    borderColor: `${colors.primaryMed}`,
+  },
+  ':valid': {
+    maxWidth: 200,
+    minWidth: 100,
+  },
+})
+const glamorousSearch = (props, {animations}) => ({
+  ...animations.expansion('width'),
+})
+const SearchBox = glamorous.input(
+  'algolia_searchbox',
+  inputBase,
+  glamorousSearch,
+)
+SearchBox.defaultProps = {
+  type: 'text',
+  pattern: '.*',
+  required: true,
+  placeholder: 'Search…',
```

Comment:
We'll need this to be localized :) Put it in `components/content/nav.md` and then you can do: `placeholder: content.search`